<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pb10">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>223</x>
     <y>52</y>
     <width>421</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb50">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb100">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>370</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb500">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>460</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCoffee">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Coffee(100)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbTea">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>320</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Tea(150)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMilk">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Mlik(200)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbReset">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>250</y>
     <width>161</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbHello">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hello</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
